Automattic/jetpack
# Admin Menu: Update Jetpack logo SVG and menu SVG icon styling

Related: Automattic/wp-calypso#49145

#### Changes proposed in this Pull Request:
The current SVG had the inner triangles set as white polygons. This
version is a single path with the triangles cut out, so that it will sit
better on any background colour.

#### Jetpack product discussion


Not applicable, this is part of the nav unification project

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
This icon is only used on WPCOM currently, so this branch needs to be used on a test Atomic site, or using the generated diff.

- Add [the Jetpack beta plugin](https://jetpack.com/download-jetpack-beta/) to an Atomic site
- In the Jetpack Beta settings, search for this feature branch (update/admin-menu-logo) and activate it
- Check that the updated Jetpack logo is acceptable.

Before | After
------ | -----
<img width="272" alt="Screen Shot 2021-04-05 at 4 07 06 PM" src="https://user-images.githubusercontent.com/1689238/113621003-15e3ac80-9629-11eb-8bf0-a08c3621f207.png"> | <img width="272" alt="Screen Shot 2021-04-05 at 4 07 40 PM" src="https://user-images.githubusercontent.com/1689238/113621002-154b1600-9629-11eb-8289-ea703fdc426e.png">
